index.html

```diff
@@ -0,0 +1,144 @@
+                        </svg>
+                    </a></li>      
+                </ul>
+            <button class="about-btn" type="submit">See my proejct</button>
+            </div>
+            <!-- My Skills -->
+            <div class="skills">
+                <ul>
+                    <li class="p">Languages
+                        <img src="img/Union.svg" alt="counter" class="counter">
+                        <ul class="lang">
+                            <li class="skill"><img src="img/javascript.svg" alt="">JavaScript </li>
+                            <li class="skill"><img src="img/html.svg" alt="">HTML</li>
+                            <li class="skill"><img src="img/css.svg" alt="">CSS</li>
+                        </ul>
+                    </li>
+                    <li class="p">Frameworks <img src="img/Union-right.svg" alt="counter" class="counter"></li>
+                    <li class="p hr">Skills <img src="img/Union-right.svg" alt="counter" class="counter"></li>
+                </ul>
+                </div>
```

Comment:
- Let's kindly adjust our styling to match the Figma design as much as we can as your design is not consistent with the Figma design. please, kindly check the Figma template to update your style


*your styling*
![Screenshot from 2022-12-06 23-19-42](https://user-images.githubusercontent.com/59397159/206036806-73d1f065-114a-4af2-9de8-5c470401a4b9.png)

Reply:
Hi, zieeco
when I can't push my updates.

 
![error](https://user-images.githubusercontent.com/51516029/206447481-5fce3f76-6a84-496b-bbb6-ceef49f9f5d1.png)